rotkehlchen/tests/unit/decoders/test_aave.py

```diff
@@ -216,3 +223,668 @@ def test_aave_eth_withdraw_v1(database, ethereum_inquirer):
+                    hexstring_to_bytes('0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000002715273613632226985186221669179813245119'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000006b44ba0a126a2a1a8aa6cd1adeed002e141bcd44'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbevmtx = DBEvmTx(database)
+    with dbevmtx.db.user_write() as cursor:
+        dbevmtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
+    decoder = EthereumTransactionDecoder(
+        database=database,
+        ethereum_inquirer=ethereum_inquirer,
+        transactions=eth_transactions,
+    )
+    with dbevmtx.db.user_write() as cursor:
+        events = decoder.decode_transaction(cursor, transaction=transaction, tx_receipt=receipt)
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO, usd_value=ZERO),
+            location_label='0x2715273613632226985186221669179813245119',
+            notes='Burned 0 ETH for gas',
+            counterparty=CPT_GAS,
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=26,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_WETH,
+            balance=Balance(amount=FVal('5504.540904812179621204')),
+            location_label='0x6B44ba0a126a2A1a8aa6cD1AdeeD002e141Bcd44',
+            notes='Withdraw 5504.540904812179621204 eip155:1/erc20:0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 from AAVE V2 to 0x6B44ba0a126a2A1a8aa6cD1AdeeD002e141Bcd44',  # noqa: E501
+            counterparty='aave-v2',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=27,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=EvmToken('eip155:1/erc20:0x030bA81f1c18d280636F32af80b9AAd02Cf0854e'),
+            balance=Balance(amount=FVal('5504.540904812179621204')),
+            location_label='0x2715273613632226985186221669179813245119',
+            notes='Send 5504.540904812179621204 aWETH from 0x2715273613632226985186221669179813245119 to 0x0000000000000000000000000000000000000000',  # noqa: E501
```

Comment:
Instead of this message in maker (please check it) we have something like

`Repay ...`